packages/composite-checkout/src/components/payment-request-button.js

```diff
@@ -37,7 +37,7 @@ export default function PaymentRequestButton( {
 	if ( formStatus === 'submitting' ) {
 		return (
 			<Button disabled fullWidth>
-				{ localize( 'Completing your purchase', { context: 'Loading state on /checkout' } ) }
+				{ __( 'Completing your purchase' ) }
```

Comment:
I felt that maybe that context was not that useful (how many other interpretations of "Completing your purchase" are there?), but I could be convinced otherwise.